lib/heuristics/scheduling_heuristic.rb

```diff
@@ -303,13 +304,41 @@ def update_route(route_data, first_index, first_start = nil)
       route
     end
 
+    def exist_possible_first_route_according_to_same_point_day(service_id, point_id)
+      # TODO : eventually consider unavailable_days here
+      return true unless @unlocked.include?(service_id) || @services_data[service_id][:raw].visits_number == 1
```

Comment:
Did you want to return true if visits_number > 1 ?

Reply:
you are right, I wrote it wrong here !